src/app.js

```diff
@@ -1,6 +1,79 @@
 // source goes here
-const app = function() {
-  console.log('hello from app!');
-};
+const titleEl = document.querySelector('#title')
+const tileContainerEl = document.querySelector('#numbers-container')
 
-export default app;
+export const generateNumbers = () => { // generates specified grid number of items
+  const numberOfGridItems = 144
+  return Array.from(Array(numberOfGridItems).keys()).map(num => ++num)
+}
+
+const allNumbers = generateNumbers()
+const HIGHLIGHTED = "highlighted"
+
+const tileGenerator = num => {
+  let tileEl = document.createElement('div')
+  tileEl.dataset.number = num
+  tileEl.innerText = num
+  tileEl.className = "number-tile"
+  return tileEl
+}
+
+const generateAllTiles = numberList => {
+  return numberList.map(num => tileGenerator(num))
```

Comment:
Could just pass the tileGenerator function rather than an anonymous arrow function.